doc/sphinx/proofs/writing-proofs/proof-mode.rst

```diff
@@ -742,8 +708,29 @@ Set Bullet Behavior
    - "None": this makes bullets inactive.
    - "Strict Subproofs": this makes bullets active (this is the default behavior).
 
-Modifying the order of goals
-````````````````````````````
+.. cmd:: Unfocused
```

Comment:
Can you add an "Other focusing commands" or whatever section so that they do not end up inside the "Set Bullet Behavior" section?